hardhat.config.ts

```diff
@@ -1,8 +1,50 @@
 import '@nomiclabs/hardhat-ethers';
 import '@nomiclabs/hardhat-waffle';
 import '@typechain/hardhat';
+import * as ethers from 'ethers';
```

Comment:
This is a strange syntax for importing from ethers, would normally be just import {ethers} from 'ethers'